frontend/src/components/SlideSearchFunction.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./SlideSearchFunction.css";
+import GenreList from "./GenreList";
+
+class SlideSearchFunction extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searching: true
+    };
+  }
+  render() {
+    return (
+      <div>
+        <button //button a placé dans le top bar
+          className="searchBarButton"
+          onClick={event => {
+            let newSearching = !this.state.searching;
+            this.setState({ searching: newSearching });
+          }}
+        >
+          ?
```

Comment:
Why this question mark ?

Reply:
That's the contain in my button search, it will change when I'll can add my part to the searchbutton in the topbar